### PR TITLE
Bump mapr to 0.3.0a1, install gallery

### PR DESCRIPTION
**Pull request by will-moore**
To allow testing of https://github.com/ome/omero-mapr/pull/46 with https://github.com/ome/omero-gallery/pull/25

### CODE REVIEW

**Comment by sbesson** [comment]
Successfully deployed on `test64`

```
[sbesson@test64-proxy ~]$ curl http://localhost/mapr/api/config/
{"gene": {"default": ["Gene Symbol"], "all": ["Gene Symbol", "Gene Identifier"], "ns": ["openmicroscopy.org/mapr/gene"], "label": "Gene", "case_sensitive": true}, "genesupplementary": {"default": [], "all": [], "ns": ["openmicroscopy.org/mapr/gene/supplementary"], "label": "Gene supplementary"}, "sirna": {"default": ["siRNA Identifier"], "all": ["siRNA Identifier", "siRNA Pool Identifier"], "ns": ["openmicroscopy.org/mapr/sirna"], "label": "siRNA"}, "sirnasupplementary": {"default": [], "all": [], "ns": ["openmicroscopy.org/mapr/sirna/supplementary"], "label": "siRNA supplementary"}, "phenotype": {"case_sensitive": true, "all": ["Phenotype", "Phenotype Term Accession"], "default": ["Phenotype"], "label": "Phenotype", "wildcard": {"enabled": true}, "ns": ["openmicroscopy.org/mapr/phenotype"]}, "compound": {"default": ["Compound Name"], "all": ["Compound Name"], "ns": ["openmicroscopy.org/mapr/compound"], "label": "Compound", "case_sensitive": true}, "compoundsupplementary": {"default": [], "all": [], "ns": ["openmicroscopy.org/mapr/compound/supplementary"], "label": "Compound supplementary"}, "organism": {"default": ["Organism"], "all": ["Organism"], "ns": ["openmicroscopy.org/mapr/organism"], "wildcard": {"enabled": true}, "label": "Organism"}, "antibody": {"default": ["Antibody Identifier"], "all": ["Antibody Identifier"], "ns": ["openmicroscopy.org/mapr/antibody"], "label": "Antibody", "case_sensitive": true}, "antibodysupplementary": {"default": [], "all": [], "ns": ["openmicroscopy.org/mapr/antibody/supplementary"], "label": "Antibody supplementary"}, "orf": {"default": ["ORF Identifier"], "all": ["ORF Identifier"], "ns": ["openmicroscopy.org/mapr/orf"], "label": "ORF"}, "orfsupplementary": {"default": [], "all": [], "ns": ["openmicroscopy.org/mapr/orf/supplementary"], "label": "ORF supplementary"}, "cellline": {"default": ["Cell Line"], "all": ["Cell Line"], "ns": ["openmicroscopy.org/mapr/cell_line"], "wildcard": {"enabled": true}, "label": "Cell Lines"}, "celllinesupplementary": {"default": [], "all": [], "ns": ["openmicroscopy.org/mapr/cell_line/supplementary"], "label": "Cell Lines supplementary"}, "others": {"default": ["Others"], "all": ["Others"], "ns": ["openmicroscopy.org/omero/bulk_annotations"], "label": "Others"}}[sbesson@test64-proxy ~]$ 
```

**Comment by will-moore** [comment]
@sbesson Should I open another PR to add gallery here too?

**Comment by sbesson** [comment]
@will-moore I think it makes sense to combine both apps together as they really belong to the same UI effort. In the absence of a release, you should be able to point at your branch directly using the [Git Pypi support](https://pip.pypa.io/en/stable/reference/pip_install/#git)

**Comment by will-moore** [comment]
OK, do we want the master branch of gallery, or the scc merge branch (same as on web-dev-merge) ```git+git://github.com/snoopycrimecop/omero-gallery.git@master_merge_daily#egg=omero-gallery``` ?
I guess it's a trade-off between stability and being able to test before merging?

**Comment by sbesson** [comment]
Note the deployment of `test64` which will be used for today's load testing also includes the mapr integration diff until we have a proper tag i.e.

```diff
--- a/ansible/group_vars/omero-hosts.yml
+++ b/ansible/group_vars/omero-hosts.yml
@@ -175,7 +175,7 @@ omero_web_config_set:
 # Plugins and additional web configuration
 
 omero_web_apps_packages:
-- "omero-mapr==0.3.0a1"
+- "git+git://github.com/snoopycrimecop/omero-mapr.git@master_merge_daily#egg=omero-mapr"
 - "omero-iviewer==0.6.0"
 - "git+git://github.com/snoopycrimecop/omero-gallery.git@master_merge_daily#egg=omero-gallery"
 omero_web_apps_names:
```

**Comment by sbesson** [comment]
@will-moore `omero-mapr==0.3.0a2` and `omero-gallery==3.2.0a2` have been released reflecting the state of today's testing.

**Comment by joshmoore** [comment]
Anything tagged and on pypi should work fine.

File: ansible/group_vars/omero-hosts.yml [hunk 1]
```diff
@@ -174,11 +175,13 @@ omero_web_config_set:
 # Plugins and additional web configuration
 
 omero_web_apps_packages:
-- "omero-mapr==0.2.3"
+- "omero-mapr==0.3.0a2"
 - "omero-iviewer==0.6.0"
+- "git+git://github.com/snoopycrimecop/omero-gallery.git@master_merge_daily#egg=omero-gallery"
```
Comment (manics) [inline ansible/group_vars/omero-hosts.yml]:
```suggestion
- omero-gallery==3.2.0a3
```

File: ansible/group_vars/omero-hosts.yml [hunk 2]
```diff
@@ -411,3 +414,61 @@ omero_web_apps_config_set:
...
+  - "cellline"
+  - "gene"
+  - "phenotype"
+  - "sirna"
+  omero.web.gallery.super_categories:
+    cell:
+      label: "Cell - IDR"
+      title: "Welcome to Cell-IDR"
+      query: "Sample Type:cell"
+      image: "https://idr.openmicroscopy.org/webgateway/render_image/122770/0/0/"
```
Comment (manics) [inline ansible/group_vars/omero-hosts.yml]: Will a relative URL work (`/webgateway/render_image/122770/0/0/`) or does it have to be absolute?

File: ansible/group_vars/omero-hosts.yml (hunk 1 above)
Comment (will-moore) [reply ansible/group_vars/omero-hosts.yml]: I'm now manually installing via ```pip install git+git://github.com/snoopycrimecop/omero-gallery.git@master_merge_daily#egg=omero-gallery``` so I guess that's fine by me.

File: ansible/group_vars/omero-hosts.yml (hunk 2 above)
Comment (will-moore) [reply ansible/group_vars/omero-hosts.yml]: It's placed into the html verbatim, so a relative URL would work.

**Comment by manics** [comment]
Deployed to test64 with https://github.com/will-moore/deployment/pull/1

**Pull request merged**